service/src/main/resources/db/migration/V349__preshool_and_club_term_breaks.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE preschool_term ADD COLUMN term_breaks daterange[];
+ALTER TABLE club_term ADD COLUMN term_breaks daterange[];
```

Comment:
multirange

Reply:
check constraints:
- breaks inside term
- not infinite ranges
- check assistance factor constraints